context/app/static/js/components/workspaces/LaunchWorkspaceDialog/hooks.ts

```diff
@@ -111,6 +113,22 @@ function useLaunchWorkspaceDialog() {
     ],
   );
 
+  const handleLaunch = useCallback(
+    (currentWorkspace: MergedWorkspace) => {
+      setWorkspace(currentWorkspace);
+
+      if (currentWorkspace.jobs.length > 0) {
```

Comment:
If the intent is to check if the current workspace is running can we use the earlier defined `runningWorkspaceIsCurrentWorkspace`?

Reply:
`runningWorkspaceIsCurrentWorkspace` doesn't check the given workspace parameter, but I think the naming here is a bit misleading - I'll change that and use the `isRunningWorkspace` util.